render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/solvers/affine/AffineAlignBlockWorker.java

```diff
@@ -859,16 +863,17 @@ protected void solve(
 	{
 		final BlockData<AffineModel2D, FIBSEMAlignmentParameters<M, S>> blockData = solveItem.blockData();
 		final ResultContainer<AffineModel2D> blockResults = blockData.getResults();
-		final PreAlign preAlign = blockData.solveTypeParameters().preAlign();
+		final FIBSEMAlignmentParameters<M, S> alignmentParameters = blockData.solveTypeParameters();
+		final PreAlign preAlign = alignmentParameters.preAlign();
 
-		final List<Integer> blockOptimizerIterations = blockData.solveTypeParameters().blockOptimizerIterations();
-		final List<Integer> blockMaxPlateauWidth = blockData.solveTypeParameters().blockMaxPlateauWidth();
+		final List<Integer> blockOptimizerIterations = alignmentParameters.blockOptimizerIterations();
+		final List<Integer> blockMaxPlateauWidth = alignmentParameters.blockMaxPlateauWidth();
 		final double blockMaxAllowedError = this.blockData.solveTypeParameters().blockMaxAllowedError();
 
 		final TileConfiguration tileConfig = new TileConfiguration();
 
 		// new HashSet because all tiles link to their common group tile, which is therefore present more than once
-		tileConfig.addTiles( new HashSet<>( solveItem.tileToGroupedTile().values() ) );
+		tileConfig.addTiles(new HashSet<>(solveItem.tileToGroupedTile().values()));
 
 		if (LOG.isInfoEnabled()) {
```

Comment:
By the way ... this is an example of where I think an `is<>Enabled` check is warranted.